dbt_sugar/core/task/doc.py

```diff
@@ -158,9 +149,37 @@ def orchestrate_model_documentation(self, model_name: str, columns_sql: List[str
         not_documented_columns = self.get_not_documented_columns(content, model_name)
         self.document_columns(not_documented_columns)
 
-        self.update_column_descriptions(self.columns_to_update)
+        # Will remove from the update the test that doesn't run correctly.
+        self.check_tests(schema, model_name)
+        # Method to update the description test and tags from the model.
+        self.update_model_description_test_tags(path, model_name, self.column_update_payload)
+        # Method to update the descriptions in all the schemas.yml
+        self.update_column_descriptions(self.column_update_payload)
+
         return 0
 
+    def check_tests(self, schema: str, model_name: str) -> None:
+        """
+        Method to run and add test into a schema.yml, this method will:
+
+        Run the tests and if they have been successful it will add them into the schema.yml.
```

Comment:
```suggestion
        - Run the tests against the database
        - check whether they are passing 
        - if so, it will add them to the schema.yml.
```